preconfigured-packages.json

```diff
@@ -31,7 +31,7 @@
         }
       },
       "linux": {
-        "package": "ffmpeg-cloud[core,avcodec,avdevice,avfilter,avformat,avresample,swscale,swresample,mp3lame,ffmpeg,ffprobe,x264,openssl,mjpeg,png,zlib]",
+        "package": "ffmpeg-cloud[core,avcodec,avdevice,avfilter,avformat,avresample,swscale,swresample,mp3lame,ffmpeg,ffprobe,openssl,mjpeg,png,zlib,nvcodec,fdk-aac,x264,x265,dav1d,vpx,nonfree,gpl,version3]",
```

Comment:
Note- I looked these up to see what they all were:
- **nvcodec**: Nvidia video decoding/encoding acceleration
- **fdk-aac**: AAC de/encoding via libfdk-aac, **including GPL-incompatible patent-encumbered HE-AAC**. If you do not require HE-AAC, use the built-in FFmpeg AAC codec.
- **x264**: H.264 encoding via x264
- **x265**: HEVC encoding via x265
- **dav1d**: AV1 decoding via libdav1d
- **vpx**: VP8 and VP9 de/encoding via libvpx
- **nonfree**: Allow use of nonfree code, the resulting libs and binaries will be unredistributable
- **gpl**: Allow use of GPL code, the resulting libs and binaries will be under GPL
- **version3**: Upgrade (L)GPL to version 3

Reply:
For what it's worth, "[There is no license fee required for using an AAC codec such as FDK AAC,[[19]](https://en.wikipedia.org/wiki/Fraunhofer_FDK_AAC#cite_note-19) only for manufacturing an "end-user" [codec](https://en.wikipedia.org/wiki/Codec) such as producing a phone loaded with AAC software](https://en.wikipedia.org/wiki/Fraunhofer_FDK_AAC#cite_note-20)".

However I'll investigate removing it.